CSS/main.css

```diff
@@ -0,0 +1,168 @@
+.main {
+  width: 100%;
+  grid-area: main;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: repeat(10, 1fr);
+  background: linear-gradient(90deg, rgba(255, 192, 203, 0.681), white);
+}
+.nav {
+  display: none;
+}
+.nav ul li:hover {
+  color: black;
+  transform: scale(1.2);
+}
+.hero {
+  grid-column: 2/12;
+  grid-row: 2/6;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto;
+  grid-row-gap: 20px;
+}
+.hero h1 {
+  grid-column: 1;
+  grid-row: 1/2;
+}
+.hero p {
+  grid-column: 1;
+  grid-row: 2/3;
+  font-size: 5vw;
+}
+
+.hero_img {
+  grid-column: 1;
+  grid-row: 3/4;
+}
+.hero_img img {
+  width: 100%;
+}
+.hero_text {
+}
+.products {
+  grid-column: 2/12;
+  grid-row: 7/10;
+  display: grid;
+  grid-template-columns: repeat(4, fr);
+  grid-template-rows: repeat(2, fr);
+  grid-gap: 20px 20px;
+}
```

Comment:
Nice use of grid-layout. Consider using the `grid-template-areas` and `grid-area` to make your CSS more semantic and more manageable .